src/AFQMC/HamiltonianOperations/tests/test_hamiltonian_operations.cpp

```diff
@@ -23,9 +23,6 @@
 #undef APP_ABORT
 #define APP_ABORT(x) {std::cout << x <<std::endl; exit(0);}
 
-#include <sys/stat.h>
-#include <unistd.h>
```

Comment:
PR says remove sys/stat.h. Why are you removing the other headers?

Reply:
I don't see a good reason to keep these lines. So just clean them on the way.
They are C or OS related headers. So I prefer removing them unless there is a real need.

Reply:
@markdewing you introduced these .h file inclusion. Do you remember why they are needed when first introduced?

Reply:
I think I'm listed in 'blame' because I reorganized some large check-in of the AFQMC code.   I don't think I originated it.

Reply:
@markdewing thanks for the confirmation that there is no specific reason to keep them.